gradio/chat_interface.py

```diff
@@ -112,6 +112,7 @@ def __init__(
             theme=theme,
             js=js,
             head=head,
+            fill_vertical_space=True,
```

Comment:
This should definitely be a parameter in `ChatInterface` to allow users to opt-out